23/README.md

```diff
@@ -0,0 +1,280 @@
+```
+shortname: BEP-23
+name: Performance Study: Analysis of Transaction Throughput in a BigchainDB Network
+type: Informational
+status: Raw
+editor: Alberto Granzotto <alberto@bigchaindb.com>
+contributors: Troy McConaghy <troy@bigchaindb.com>, Muawia Khan <muawia@bigchaindb.com>
+```
+
+# Performance Study: Analysis of Transaction Throughput in a BigchainDB Network
+
+## Abstract
+Transaction throughput is a key metric to evaluate the suitability of a blockchain system for real world applications. BigchainDB is now in its beta phase, and we don't expect major changes in the parts of the code that are responsible for the throughput of the system (i.e. the validation logic), for this reason we can now run performance tests and publish our results. The tested setup consists of a 4 node network running a [recent version of BigchainDB][bdb:git-commit]. Each node runs on a different virtual machine, hosted in the [Azure infrastructure][azure:landing-page]. A fifth virtual machine is used to generate the workload and collect metrics. Depending on the configuration of the system, a BigchainDB network under constant load finalizes between 300 and 1000 transactions per second.
```

Comment:
The last sentence of this abstract is the "money quote" so we need to make sure it's absolutely accurate. I'd change it slightly, to say:

Depending on the configuration of the system, the tested BigchainDB network, under constant load, finalizes between 300 and 1000 transactions per second.